cli/bpmetadata/types_ui.go

```diff
@@ -0,0 +1,124 @@
+package bpmetadata
+
+// BlueprintUI is the top-level structure for holding UI specific metadata
+type BlueprintUI struct {
```

Comment:
Let's add some kind of version info similar to standard

Reply:
There is a `version` field to track that.

Reply:
`BlueprintUI` the top most struct will be part of `BlueprintMetadataSpec` and will inherit the `Kind` and `APIVersion` from it. So there's no need to define it again here. I think I forgot to add `BlueprintUI` to `BlueprintMetadataSpec` in code which may have caused confusion. Will do that now.